products/epinio/models/services.js

```diff
@@ -8,16 +8,16 @@ import EpinioResource from './epinio-resource';
 export default class EpinioService extends EpinioResource {
   get links() {
     return {
-      update: this.getUrl(),
-      self:   this.getUrl(),
-      remove: this.getUrl(),
-      create: this.getUrl(this.meta?.namespace, null), // ensure name is null
+      update: this.getUrl(this.meta?.namespace, this.meta?.name),
+      self:   this.getUrl(this.meta?.namespace, this.meta?.name),
+      remove: this.getUrl(this.meta?.namespace, this.meta?.name),
+      create: this.getUrl(this.meta?.namespace, ''),
     };
   }
 
-  getUrl(namespace = this.meta?.namespace, name = this.meta?.name) {
-    // Add baseUrl in a generic way
-    return this.$getters['urlFor'](this.type, this.id, { url: `/api/v1/namespaces/${ namespace }/services/${ name || '' }` });
+  getUrl(namespace = this.meta?.namespace) {
+    // Don't include the name when creating a service.
+    return this.$getters['urlFor'](this.type, this.id, { url: `/api/v1/namespaces/${ namespace }/services/${ name }` });
```

Comment:
name is not defined

Reply:
The problem with using a default value for the service name was that it was using the service name when trying to create it. Using a default name gave it the incorrect route to post to.